src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/Example2.java

```diff
@@ -0,0 +1,63 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="Indentation">
+      <property name="caseIndent" value="0"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;
+
+// xdoc section -- start
+class Example2 {
+    String field = "example";// basicOffset
+    int[] values = {// basicOffset
```

Comment:
I likes how it was before, when comments are vertically aligned by same column.

Reply:
done